### PR TITLE
Joss Review Suggestions

### PR DESCRIPTION
**Pull request by cxhernandez**
re: #77

__Installation:__
- [x] Support conda install with python 3.6. 
- [x] Specify version requirements (if applicable) for dependencies. The Fs-peptide notebook gets angry at pandas 0.16 (a few years behind the times) for the tICA cell. Alternatively, suggest that users update to most recent version for these dependencies if they are using an existing python 3.4 build.

__Example usage:__
- [x] Describe a possible use case for the chord diagram. It seems a bit disconnected from the rest of the tools, so adding an example (perhaps to some visualization of FS-peptide structure) could really tie things together.

__Community guidelines:__
- [x] Add a sentence to the "Development" section of your README directing potential contributors to the link you have already included.
- [x] Include a preferred method of contact for users seeking support.

__Version:__
- [ ] Cut v1.0.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,9 +122,18 @@ You may instead want to use the development version from Github, by running
 Development
 -----------
 
-https://github.com/msmexplorer/msmexplorer
+All development happens here, on
+(Github)[https://github.com/msmexplorer/msmexplorer].
```
Comment (mpharrigan) [inline README.md]: markdown link style backwards! `[github](linky.html)`

**Comment by mpharrigan** [comment]
Cool!

File: README.md (hunk 1 above)
Comment (cxhernandez) [inline README.md]: whoops! so sleepy.... 😴

**Comment by msultan** [comment]
Seems fine to me but as do you need to add ```msmbdata``` as a dependency as well since you removed the code that gets the data?

**Comment by cxhernandez** [comment]
good point!

**Comment by cxhernandez** [comment]
MSMBuilder requires it already (as evidenced by the tests working 😄 )

**Pull request merged**